nvflare/fuel/utils/json_config_loader.py

```diff
@@ -0,0 +1,144 @@
+    def __init__(self, conf: Dict, file_path: Optional[str] = None):
+        self.conf = conf
+        self.format = ConfigFormat.JSON
+        self.file_path = file_path
+
+    def get_conf(self):
+        return self.conf
+
+    def get_format(self):
+        return self.format
+
+    def get_location(self) -> Optional[str]:
+        return self.file_path
+
+    def to_dict(self) -> Dict:
+        return self.conf
+
+    def to_conf(self, element: Dict) -> str:
+        return json.dumps(element)
+
+    def get_int(self, key: str, default=None) -> Optional[int]:
+        value = self.conf.get(key, default)
+        try:
+            return int(value) if value is not None else None
+        except (TypeError, ValueError):
+            raise ConfigError("{key} has type '{type}' rather than 'int'".format(key=key, type=type(value).__name__))
+
+    def get_float(self, key: str, conf=None, default=None) -> Optional[float]:
+        value = self.conf.get(key, default)
+        try:
+            return float(value) if value is not None else None
+        except (TypeError, ValueError):
+            raise ConfigError("{key} has type '{type}' rather than 'float'".format(key=key, type=type(value).__name__))
+
+    def get_bool(self, key: str, default=None) -> Optional[bool]:
+        v = self.conf.get(key, default)
+
+        if v is None:
+            return v
+        if isinstance(v, bool):
+            return v
+        if isinstance(v, int):
+            return v != 0
+        if isinstance(v, str):
+            v = v.lower()
+            return v in ["true", "t", "yes", "y", "1"]
+        raise ConfigError(f"{key} has type '{type(v).__name__}' value '{v}' cannot be converted to bool ")
+
+    def get_str(self, key: str, default=None) -> Optional[str]:
+        value = self.conf.get(key, default)
+        try:
+            return str(value) if value is not None else None
+        except (TypeError, ValueError):
+            raise ConfigError(
+                "{key} has type '{type}' cannot covert to 'str'".format(key=key, type=type(value).__name__)
+            )
+
+    def get_list(self, key: str, default=None) -> Optional[List]:
+        value = self.conf.get(key, default)
+        if value:
+            if isinstance(value, List):
+                return value
+            else:
+                raise ConfigError(
+                    "{key} has type '{type}' rather than 'List'".format(key=key, type=type(value).__name__)
+                )
+        else:
+            return None
+
+    def get_config(self, key: str, default=None):
+        value = self.conf.get(key, default)
+        if value:
+            if isinstance(value, Dict):
+                return JsonConfig(value, self.file_path)
+            else:
+                raise ConfigError(
+                    "{key} has type '{type}' rather than 'Dict' or 'JsonConfig' ".format(
+                        key=key, type=type(value).__name__
+                    )
+                )
```

Comment:
if value is Dict, things are good,
if value is of type "JsonConfig", this code will raise ConfigError